### DISCUSSION TITLE
Docker Crash When Updating From 1.91.0 -> 1.91.1

### DISCUSSION THREAD

**Discussion by unnerving-sprinkler** [immich-app/immich#5760] [Q&A] [answered]
Maybe this is a Docker question more than an immich question, but when attempting to update from 91.0 to 91.1 the machine learning line will sit and spin for a couple of minutes before crashing docker and throwing an unexpected EOF. 

![image](https://github.com/immich-app/immich/assets/32282786/730b9bb5-4f9b-4eaf-8b76-092ac1d1f786)

**Comment by alextran1502** [comment]
Check if your disk space is still enough to pull the images

**Reply by unnerving-sprinkler** [reply]
There appears to be plenty of space.

![image](https://github.com/immich-app/immich/assets/32282786/7e0c9b54-d679-4b49-9e25-eef811949b20)

I am running this on an Ubuntu Proxmox VM, 4C 3GB ram, usually no issues with running immich in this config.

**Reply by alextran1502** [reply, marked as answer]
I would be 4-5GB of RAM just to be sure, but it is not the related to the problem we are solving here.

If you try pulling again, does it work? 

ALso try to restart docker engine with `sudo systemctl restart docker`

**Reply by unnerving-sprinkler** [reply]
Set ram to 5GB & restarted docker without really any luck. I ended up removing the container and manually deleted all the images then ran `docker compose up -d`, which worked. 

I'll try the 'normal' method again on the next update. I changed around how much of the system memory Docker can use, I think docker might have been ballooning and then crashing. Set it back down so docker can only eat 4/6GB and 3/4 cores. Should leave enough overhead for Ubuntu even if docker gets hungry. 

Thanks @alextran1502
